docs/posts/contribute.md

```diff
@@ -0,0 +1,70 @@
+# Code of conduct
```

Comment:
Since a lot of this is just a rehash of CONTRIBUTING.md, it may be more beneficial to link to `https://github.com/chanzuckerberg/cellxgene/blob/master/CONTRIBUTING.md` and move these changes there.  This will prevent keeping these two files in sync and prevent redundancy.

Reply:
Yeah. I'm trying to move everything to the same place; if no one objects, I'll actually just change the `contributing.md` file (and other similar duplications) to point to the main docs.